docs/imviz/displayimages.rst

```diff
@@ -5,91 +5,142 @@ Displaying Images
+
+    :ref:`Pan/Zoom <cubeviz-pan-zoom>`
+    Documentation on using Pan/Zoom in the Jdaviz viewers.
+
+From the API, you can programmatically zoom in and out. Zoom level:
+
+    * 1 - real-pixel-size
+    * 2 - zoomed in by a factor of 2
+    * 0.5 - zoomed out by a factor of 2
+    * 'fit' - zoomed to fit the whole image width into display
+
+For example::
+
+    viewer = imviz.default_viewer
+    viewer.zoom_level
+    viewer.zoom_level = 1  # Set the zoom level directly.
+    viewer.zoom(2)  # Set the relative zoom based on current zoom level.
 
 .. _imviz_defining_spatial_regions:
 
 Defining Spatial Regions
 ========================
 
-Spatial regions can be created the same way as in Cubeviz.
+Spatial regions allow users to select subsets of the data array for
+specific analysis function in the plugin toolbar, such as aperture photometry.
+Users can create spatial regions either in Imviz or the Jupyter notebook.
 
 .. seealso::
 
     :ref:`Defining Spatial Regions <spatial-regions>`
     Documentation on defining spatial regions in an image viewer.
 
+.. seealso::
+
+    :ref:`Importing Spatial Regions <imviz-import-regions-api>`
+    Importing spatial regions from within the Jupyter notebook.
+
+You can :ref:`import regions from the API<imviz-import-regions-api>`.
+You can also retrieve the results as `regions.Regions` as follows, assuming ``imviz`` is the instance of your Imviz application::
+
+    regions = imviz.get_interactive_regions()
+    regions
+
 Blinking
 ========
 
 Blinking is an Imviz-specific functionality that allows a user to quickly switch
-between viewing two or more images, as long as they are linked
-(see :ref:`imviz_pan_zoom` for more on linking behavior). This can be done
-by selecting the |icon-blink| icon and then clicking on the image.
-You can also blink by selecting the "b" key on your keyboard while moused over the image.
+between viewing two or more images, as long as they are linked (see :ref:`imviz_pan_zoom` for
+more on linking behavior). This can be done by selecting the |icon-blink| icon and
+then clicking on the image. You can also blink by selecting the "b" key on your
+keyboard while moused over the image.
+
+From within the Jupyter notebook::
+
+    viewer = imviz.default_viewer
+    viewer.blink_once()
 
 Contrast/Bias
 =============
 
 In addition to changing :ref:`contrast` and :ref:`bias` in the :ref:`display-settings`,
 Imviz has a |icon-white-to-black| button under the |icon-blink| menu that can also
 adjust those values.
+
 After right-clicking on the blink icon, left click on the constrast/bias icon to activate it.
 Now you can click and drag on the image viewer to change to change the contrast
 and bias. Moving along the X-axis will change the bias and moving along the Y-axis will change the
 contrast. If you would like to reset to the default contrast and bias settings, you can
 double-click on the display while the mode is active.
 
-.. _imviz_cursor_info:
-
-Cursor Information
```

Comment:
I don't know why this is removed, so I am adding it back it but higher up the page.